chainercv/utils/image_utils.py

```diff
@@ -2,22 +2,23 @@
 from PIL import Image
 
 
-def read_image_as_array(path, dtype=np.uint8, copy=True):
+def read_image_as_array(path, dtype=np.uint8, copy=True, force_color=False):
```

Comment:
I think the dafault value of `force_color` should be `True`.
This is because gray scale images are expected to be arrays of shape (3, H, W) and not (1, H, W), and this should be the default behavior.